membership-attribute-service/app/configuration/Config.scala

```diff
@@ -49,6 +49,13 @@ object Config {
   val mmaCorsConfig = CORSConfig.denyAll.copy(
     allowedOrigins = config.getStringList("mma.cors.allowedOrigins").asScala.toSet
   )
+
+  lazy val mmaCardCorsConfig = Config.mmaCorsConfig.copy(
+    isHttpHeaderAllowed = Seq("accept", "content-type", "csrf-token").contains(_),
```

Comment:
turns out the cors filter does some normalisation after all so the case is correct